### PR TITLE
Gather one week of node logs and drop audit logs (RFE-309)

### PR DESCRIPTION
**Pull request by sferich888**
By dropping audit log collection and providing a way to set timeframe, with a default

### CODE REVIEW

**Comment by sferich888** [comment]
@wking @deads2k @sanchezl can you all please review.

**Comment by wking** [comment]
> until we get openshift/release#7596...

Landed.

/hold cancel

**Comment by wking** [comment]
/retitle Gather one day of node logs and drop audit logs (RFE-309)

Do we want to document how to gather audit logs when you do want them somewhere?

e2e gather looks good:

```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_must-gather/143/pull-ci-openshift-must-gather-master-e2e-aws/145/artifacts/e2e-aws/must-gather.tar | tar -tz | grep 'audit_logs\|service.log'
./registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/host_service_logs/
./registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/host_service_logs/masters/
./registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/host_service_logs/masters/crio_service.log
./registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/host_service_logs/masters/kubelet_service.log
```

although [the job][1] failed with:

```
fail [github.com/openshift/origin/test/extended/cli/mustgather.go:108]: Expected
    <string>: /tmp/test.oc-adm-must-gather.553366366/registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/audit_logs/kube-apiserver
to be a directory: stat /tmp/test.oc-adm-must-gather.553366366/registry-svc-ci-openshift-org-ci-op-7mgmn27y-stable-sha256-072c6ddb132f67d9912b44e9870f4d5b6168155232c6c9f051cc37320a34e514/audit_logs/kube-apiserver: no such file or directory
```

Isn't the lack of `audit_logs` expected with this change?

[1]: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_must-gather/143/pull-ci-openshift-must-gather-master-e2e-aws/145

**Comment by wking** [comment]
Ah, I'll PR origin to separate the audit-log tests...

/hold

**Comment by wking** [comment]
openshift/origin#24680 should, if/when it lands, get e2e-aws passing again for this PR.

**Comment by deads2k** [comment]
/lgtm
/hold

holding for https://github.com/openshift/origin/pull/24680 which doesn't compile.

**Comment by wking** [comment]
/retitle Gather one week of node logs and drop audit logs (RFE-309)

Catching up with [this][1].

[1]: https://github.com/openshift/must-gather/pull/143#event-3124458506

**Comment by soltysh** [comment]
/retest
to see where we are with this

**Comment by wking** [comment]
/hold cancel
/retest

openshift/origin#25283 landed 🎊🎉

File: collection-scripts/gather [hunk 1]
```diff
@@ -45,4 +45,4 @@ done
 /usr/bin/gather_audit_logs
 
 # Gather Service Logs (using a suplamental Script); Scoped to Masters.
-/usr/bin/gather_service_logs master
+/usr/bin/gather_service_logs master
```
Comment (wking) [inline collection-scripts/gather:48]: No need to touch this file line?

File: collection-scripts/gather [hunk 2]
```diff
@@ -45,4 +45,4 @@ done
 /usr/bin/gather_audit_logs
```
Comment (wking) [inline collection-scripts/gather:45]: Drop this line?

**Comment by deads2k** [comment]
/hold

https://github.com/openshift/origin/pull/25283 is flaking badly.

**Comment by deads2k** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1861746 is open and needs resolution before merge.

**Comment by sferich888** [comment]
Closing in favor of https://github.com/openshift/must-gather/pull/144

**Pull request closed without merging**